Support custom levels (#27)
pinojs/pino-syslog: Readme.md

[tzstring]: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
[luxon]: https://moment.github.io/luxon/#/zones?id=specifying-a-zone

### Custom levels

> You can now add custom levels but you have to make sure that your new custom level is supported by pino ([docs](https://github.com/pinojs/pino/blob/HEAD/docs/api.md#opt-customlevels))

Reviewer: ```suggestion
Custom [Pino levels](https://github.com/pinojs/pino/blob/HEAD/docs/api.md#opt-customlevels) are supported.
They must be established through a mapping defined under the `customLevels`
configuration key. `customLevels` is a hash of hashes. Each key under
`customLevels` is a custom level name with a value that is a hash with
keys `level` and `syslogSeverity`. The `level` key maps to the log level number,
and the `syslogSeverity` key to the name of a spec compliant syslog level:
"emergency", "alert", "critical", "error", "warning", "notice", "info", or
"debug".
```

> The following configuration example shows how it is possible to add customized levels:

Reviewer: ```suggestion
The following example shows how to add customized levels:
```

```json
{
	"modern": true,
  	"appname": "none",
  	...
  	"customLevels": {
		"customLevel_name": {
			"level": 70, // Custom pino level.
			"syslogSeverity": "alert" // Well-known syslog severity
		}
   	}

> }

Reviewer: Please make this valid JSON and use consistent whitespace.

[tzstring]: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
[luxon]: https://moment.github.io/luxon/#/zones?id=specifying-a-zone

### Custom levels

Custom [Pino levels](https://github.com/pinojs/pino/blob/HEAD/docs/api.md#opt-customlevels) are supported.
They must be established through a mapping defined under the `customLevels`
configuration key. `customLevels` is a hash of hashes. Each key under
`customLevels` is a custom level name with a value that is a hash with
keys `level` and `syslogSeverity`. The `level` key maps to the log level number,
and the `syslogSeverity` key to the name of a spec compliant syslog level:
"emergency", "alert", "critical", "error", "warning", "notice", "info", or
"debug".

The following example shows how to add customized levels:

```json

> {
>   "modern": true,
>   "appname": "none",
>   "customLevels": {
>     "customLevel_name": {
> 	  "level": 70,
> 	  "syslogSeverity": "alert"
> 	}
>   }
> }

Reviewer: ```suggestion
{
  "modern": true,
  "appname": "none",
  "customLevels": {
    "customLevel_name": {
      "level": 70,
      "syslogSeverity": "alert"
    }
  }
}
```
